### PR TITLE
added get storage

### PR DESCRIPTION
**Pull request by wulzymart**
Pull request for adding get and count functionality to storages

### CODE REVIEW

**Review by AdekunleAdetunji — CHANGES_REQUESTED** [review]
Remove the __pycache__ binary files

**Comment by wulzymart** [comment]
changes effected

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -51,6 +51,27 @@ def all(self, cls=None):
                     new_dict[key] = obj
         return (new_dict)
 
+    def get(self, cls, id):
+        """returns a single item"""
+        if cls not in classes.values():
```
Comment (AdekunleAdetunji) [inline models/engine/db_storage.py]: Remove this block, it will return None irregardless

File: models/engine/db_storage.py [hunk 2]
```diff
@@ -51,6 +51,27 @@ def all(self, cls=None):
...
+            return None
+        all_items = self.all(cls).values()
+        for item in all_items:
+            if item.id == id:
+                return item
+        return None
```
Comment (AdekunleAdetunji) [inline models/engine/db_storage.py]: Remove this line

File: models/engine/file_storage.py [hunk 3]
```diff
@@ -33,6 +33,27 @@ def all(self, cls=None):
                     new_dict[key] = value
             return new_dict
         return self.__objects
+    
+    def get(self, cls, id):
+        """returns a single item"""
+        if cls not in classes.values():
```
Comment (AdekunleAdetunji) [inline models/engine/file_storage.py]: remove this block, it will return none irregardless

File: models/engine/file_storage.py [hunk 4]
```diff
@@ -33,6 +33,27 @@ def all(self, cls=None):
...
+            return None
+        all_items = self.all(cls).values()
+        for item in all_items:
+            if item.id == id:
+                return item
+        return None
```
Comment (AdekunleAdetunji) [inline models/engine/file_storage.py]: remove this line

**Review by AdekunleAdetunji — CHANGES_REQUESTED** [review]
Please remove those lines i commented on, they are not needed.

**Review by AdekunleAdetunji — APPROVED** [review]
This looks great

**Pull request merged**